CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ include(CTest)
 enable_testing()
 #need to define the version of the library
 set(MAJOR_VERSION "9")
-set(MINOR_VERSION "2")
+set(MINOR_VERSION "3")
```

Comment:
Going from 9.2.6 to 9.3.6?

Reply:
Yep, according to [semantic versioning](https://semver.org/) I would consider this change as _MINOR version when you add functionality in a backwards-compatible manner_

Reply:
Or do you mean we should drop PATCH_VERSION to `0`?